synctl

```diff
@@ -176,8 +179,7 @@ def main():
         )
         sys.exit(1)
 
-    with open(configfile) as stream:
-        config = yaml.safe_load(stream)
+    config = read_config_files(find_config_files([configfile]))
```

Comment:
I'm not super-keen on using read_config_files here. It basically has two parts: (1) a loop, which is probably clearer to copy here, and (2) some sanity-checks on the config, which I'd prefer were left to synapse itself rather than synctl.